bower.json

```diff
@@ -39,7 +39,7 @@
     "angular-sanitize": "1.3.0 - 1.5.*",
     "angular-bootstrap": "0.13.x",
     "lodash": "3.x",
-    "patternfly": "~3.7.0"
+    "patternfly": "git://github.com/patternfly/patternfly.git#master-dist"
```

Comment:
@jeff-phillips-18 This was just temporary while I was testing last night. I'm new to our release process - should we update this reference after we publish our new Patternfly release?